dist/index.html

```diff
@@ -20,11 +20,16 @@ <h2>fitlit</h2>
       <section class="content user-profile">
         <figure>
           <img class=user-img src="./images/turing-logo.png" alt="turing logo">
-          <figcaption>User Name goes here</figcaption>
+          <figcaption id="userFullName">User Name goes here</figcaption>
+          <p>Email: <span id="userEmail">email@example.org</span></p>
+          <p>Address: <span id="userAddress">123 Street Road</span></p>
+          <p>Stride Length: <span id="userStride">3.4</span></p>
+          <p>Friends: <span id="userFriends">John Doe, Lindsey Smith</span></p>
```

Comment:
This works for now to complete the iteration, however let's figure out how to use a for loop and innerHTML to make a more engaging friends list.